interface/forms/misc_billing_options/new.php

```diff
@@ -82,7 +80,7 @@
       <label for="box10d"><?php echo xlt('Box 10. d. EPSDT Referral Code:'); ?></label>
       <input type="entry" size=2 name="medicaid_referral_code" id="box10d" value="<?php echo attr($obj{"medicaid_referral_code"}); ?>" >
       <label for="box10depsdt"><?php echo xlt('EPSDT:'); ?></label>
-      <input type="checkbox" name="epsdt_flag" id="box10depsdt" value="1" <?php if ($obj['epsdt_flag'] == "1") echo "checked"; ?> ><br><br>
+      <input type=checkbox name="epsdt_flag" id="box10depsdt" value="1" <?php if ($obj['epsdt_flag'] == "1") echo "checked"; ?> ><br><br>
```

Comment:
generally good practice to surround html attributes with quote (either single or double):
https://www.w3schools.com/html/html_form_input_types.asp